src/app/pages/jails/jail-list/jail-list.component.ts

```diff
@@ -80,7 +80,7 @@ export class JailListComponent implements OnInit {
                 }
               }
               if (message === "") {
-                this.snackBar.open(T("Jails started."), 'close', { duration: 5000 });
+                this.dialogService.Info(T('Jails Started'), T("Jails started."));
```

Comment:
I would make the second "Jails started" just be an empty '', give it an 'info' icon and set html to true